skyvern/webeye/actions/handler.py

```diff
@@ -828,9 +820,16 @@ async def normal_select(
     action: actions.SelectOptionAction,
     skyvern_element: SkyvernElement,
 ) -> List[ActionResult]:
+    try:
+        current_text = await skyvern_element.get_attr("selected")
+        if current_text == action.option.label or current_text == action.option.value:
+            return [ActionSuccess()]
+    except Exception:
+        LOG.info("failed to confirm if the select option has been done, force to take the action again.")
```

Comment:
why do we say "force to take the action again"? we're not retrying here, are we?

Reply:
em. if the skip logic failed. we try to select again!